tests/updateArticle.test.js

```diff
@@ -60,7 +60,7 @@ describe('testing the middlewares before reaching the update article controller'
       .patch(`/api/articles/${slug}`)
       .set('authorization', UserToken)
       .send({
-        body: ['how did the classical Latin become']
+        body: 'how did the classical Latin become'
```

Comment:
why make changes here?

Reply:
yeah, the body is meant to be a string, not an array of string

Reply:
so after putting some validations, it was throwing an error

Reply:
okay